pkg/dao/repository_config_test.go

```diff
@@ -996,8 +997,9 @@ func (suite *RepositoryConfigSuite) TestValidateParameters() {
 		UUID: &repoConfig.UUID,
 	}
 
-	mockExtRDao.Mock.On("FetchRepoMd", repoConfig.Repository.URL).Return(pointy.String("<XMLFILE>"), 200, nil)
-	mockExtRDao.Mock.On("FetchSignature", repoConfig.Repository.URL).Return(pointy.String("sig"), 200, nil)
+	mockExtRDao.Mock.On("FetchRepomd").Return(200, nil)
```

Comment:
Only updated this one test as a demonstration for now because I'm not confident about how this is architect-ed.